src/file-upload/file-upload.scss

```diff
@@ -79,8 +80,7 @@
     }
     visibility: visible;
 
-    > svg,
-    > svg > * {
+    svg {
```

Comment:
so no need to tackle inner elements anymore?

Reply:
See comment above about path.